### PR TITLE
Light fixes and light level tweaks for immulsions

### PR DESCRIPTION
**Pull request by ghost**
[bugfix] [tweak] [general]

I had a suspicion that something was wrong with lighting, turns out the standard wall-mounted light objects have been broken for over a year.

* Fixed some issues with lights and removed a feature that gave them variable light distance on spawn (as the values were so high that it wasn't even noticeable). 
* Tweaked light values for standard lights to be a bit moodier. No more fullbright stations. Standard tubes were given the same color as lightbulbs, while HE bulbs were properly assigned their missing LIGHT_COLOR_HALOGEN value.
* HE tubes are now as bright as regular lights were previously. They were the same before these changes, so now you actually have a reason to buy and install them.
* Replaced all botany lights (including the botany areas in permabrig and vox outpost) with HE tubes on all maps except Snaxi, since Kurfursten is working on an overhaul for it at the moment. As a result, light changes shouldn't fuck with botany light level requirements.

dreamseeker_ITQpVH7uQS
dreamseeker_CQuQWYIYFe
dreamseeker_Y2EWOf6EUO
dreamseeker_qyYsPmfKIV
dreamseeker_3d2ukpQvFS

:cl:
 * bugfix: Fixed standard lightbulbs/tubes having their light values overwritten on spawn.
 * tweak: Increased atmosphere from slightly toning down standard light brightness.
 * tweak: Replaced all botany lights with HE tubes to make up for overall lower light levels.

### CODE REVIEW

**Comment by brndd** [comment]
This leads to annoying dark spots in the middle of rooms and corridors, and previously deliberately dimly lit rooms becoming far too dim. Given the limitations of our lighting system I think a better approach would be to tweak the hue of the lights, not their range.

**Comment by Kharkov72** [comment]
I mean, unless you want to remap every light in existence.
Then by all means.
Also, personally I'm a fan of the white lights, although yellow can be atmospheric in certain situations. I like this idea, but I'm torn.

**Comment by RealestEstate** [comment]
I partially agree with brnndi, in particular: kitchen, bar, and robotics have an awkward dark spot just judging by the photos.

**Comment by ghost** [comment]
I personally don't see why the station has to be fully and perfectly lit. Particularly jarring cases of dark spots can be fixed as they're noticed in future mapping changes.  Lesser cases can be remedied by carrying a flashlight (or ugprading your PDA, which makes it as bright as a flashlight). Or you can order HE lights since they're actually better than regular ones now. There's plenty of options of dealing with darkness.

File: __DEFINES/lighting.dm [hunk 1]
```diff
@@ -65,7 +65,7 @@
 #define LIGHT_COLOR_FIRE       "#FAA019" //Warm orange color, leaning strongly towards yellow. rgb(250, 160, 25)
 #define LIGHT_COLOR_FLARE      "#FA644B" //Bright, non-saturated red. Leaning slightly towards pink for visibility. rgb(250, 100, 75)
 #define LIGHT_COLOR_SLIME_LAMP "#AFC84B" //Weird color, between yellow and green, very slimy. rgb(175, 200, 75)
-#define LIGHT_COLOR_TUNGSTEN   "#FAE1AF" //Extremely diluted yellow, close to skin color (for some reason). rgb(250, 225, 175)
+#define LIGHT_COLOR_TUNGSTEN   "#FFF5E0" //Extremely diluted yellow, close to skin color (for some reason). rgb(250, 225, 175)
```
Comment (brndd) [inline __DEFINES/lighting.dm]: Update the comment to match the new RGB values of this (255, 245, 224)

**Comment by brndd** [comment]
>I personally don't see why the station has to be fully and perfectly lit.

Because our lighting system is quite binary. There's not much gradation between being fully lit and not being lit at all. For this reason I'd prefer a far less radical range tweak (eg. 8 tiles to 7 tiles, or at least 6, instead of the 5.2 that you changed it to now) combined with a more radical colour tweak.

If the regular light tubes have a dim-feeling colour, replacing them with HE bulbs will produce a more noticeable effect, which may be enough encouragement to make people actually use HE bulbs. In fact considering HE bulbs used to be bugged and didn't change the colour of the light *at all*, it's possible that that bugfix alone will increase their use.

File: __DEFINES/lighting.dm (hunk 1 above)
Comment (ancientpower) [reply __DEFINES/lighting.dm]: ok

**Comment by ghost** [comment]
I can bump it up to 5.5 or so but 6 is when it basically becomes fullbright and it's no longer noticeable that lights even have a range except in large rooms.

**Comment by DamianX** [comment]
MAXBRIGHTNESS is bad, black spots are equally bad but they can be fixed by moving lights around where necessary.
I'm totally OK with this being merged as it is and for mapping fixes to come later.

**Comment by brndd** [comment]
But who's gonna do those mapping fixes? Mapping lights is a giant fucking pain because you can't see their radius in the mapper. You need to have the game running in the background (with roundstart blown lights turned off) and cross-compare to fix bad spots. Considering the magnitude of this change there will likely be dozens if not hundreds such spots for each map.

**Comment by Kurfursten** [comment]
That's what we have @Probe1 for

**Comment by beesil** [comment]
Updating the `brightness_range` without changing `brightness_power` will help with the dark spots.

**Comment by ghost** [comment]
I increased the range to 6 and the dark spots are significantly less noticeable. I'll post new screenshots tomorrow or something.

**Comment by FudgePucker** [comment]
What about engineering, atmospherics in particular, that room is a candidate for HE lighting anyways sense it runs out of power so fast due to distance from SMESs, and having so many machines and lights.

**Comment by ghost** [comment]
Atmospherics on box and some other maps starts with floodlights specifically for that purpose. I might add HE lights to medbay in a later PR since it would take a bit more effort and not be very atomic.

**Pull request merged**